docs/draft/notifications.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Notifications
+nav_order: 2
+has_children: true
+permalink: /specs/draft
```

Comment:
Add number to spec and PR title; make sure it doesn't conflict with existing PRs